ItemListResponse.swift

```diff
@@ -11,5 +11,5 @@ import Foundation
 struct ItemListResponse: Codable, Identifiable {
     var id = UUID()
     let item: String
-    let saved: Bool
+    var saved: Bool
```

Comment:
Made this a `var` so I could simulate the value changing in the fake network requests.